docs/getting-started.mdx

```diff
@@ -408,6 +437,68 @@ up. To run `integration_test/example_test.dart` on a connected Android, iOS or m
           internals depend on it.
       </Error>
 
+### FAQ
+
+<Accordion title="Testing fails with error inside 'integration_test/test_bundle.dart'">
+  The reason is probably a mismatch of `patrol` and `patrol_cli` versions. Make sure that you 
+  use the latest versions of both packages.
+</Accordion>
+
+<Accordion title="'example_test.dart' passed but I can't get my application to run 
+within the patrol test.">
+  To run your application within the patrol test, you need to call `$.pumpWidgetAndSettle()`,
+  and pass your application's main widget to it. Be sure that you registered all the 
+  necessary services before calling `$.pumpWidgetAndSettle()`.
+  Here's the example of running an app within the patrol test:
+  ```dart
```

Comment:
it's good practice to put an empty newline in places like this one

standard markdown formatters like Prettier would do it